src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+:root {
+  --bg-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.moyo {
+  margin: 10px 50px;
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+}
+
+.nav {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  height: 60px;
+}
+
+.navList {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  flex-wrap: nowrap;
+}
+
+.navItem {
+  position: relative;
+  margin: 0 20px 0 0;
+  padding: 0;
+  justify-content: space-between;
+}
+
+.button {
+  display: inline-block;
+  text-align: center;
+  text-decoration: none;
+  color: #000;
+  height: 100%;
+  line-height: 60px;
+  position: relative;
+  cursor: pointer;
+}
+
+.navItem:first-child a::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--bg-color);
+  border-radius: 8px;
+  opacity: 1;
+}
+
+.header .nav .navList .navItem .is-active {
+  color: var(--bg-color);
+}
```

Comment:
```suggestion

```

Reply:
Not fixed:

![image](https://github.com/user-attachments/assets/08291f5e-d163-4dd8-b227-887b78f09e9a)

Redundant code - delete it